interop/src/archconfig.ts

```diff
@@ -119,6 +119,12 @@ export interface DeviceConfig
      * If pin X (`pins.X`) is to be exposed as Y, then `$pins.Y == X`.
      */
     $pins?: Record<string, string>
+
+    /**
+     * Extra arguments to be passed to the flash tool.
+     * For example, `--flash_mode` for `esptool.py` (ESP32 boards).
+     */
+    flashToolArguments?: string[]
```

Comment:
Please rename to `$flashToolArguments` - the identifiers starting with `$` are not stored in device flash and instead only used by the CLI tooling

Reply:
I wondered what the convention there was; with some being specified and others not.